code/modules/mob/living/silicon/pai/pai_say.dm

```diff
@@ -5,4 +5,4 @@
 		..(message)
 
 /mob/living/silicon/pai/binarycheck()
-	return 0
+	return radio.translate_binary
```

Comment:
```suggestion
	return radio?.translate_binary
```
Not all silicons have radios in them, so it's good to support this check being compatible with them in case we create one pAI without one.
This would avoid a runtime.